<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Encoder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Encoder:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_EncoderSelectComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="m_SelectedEncoderGroupBox">
      <property name="title">
       <string>Encoder</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>File to encode</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="m_FileToEncodeLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_BrowseFileToEncodePushButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="m_EncodePushButton">
        <property name="text">
         <string>Encode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_EncodeAsPushButton">
        <property name="text">
         <string>Encode As...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextEdit" name="m_LogTextEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_EncoderSelectComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>OnEncoderChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_EncodePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnEncodeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_EncodeAsPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnEncodeAsButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_BrowseFileToEncodePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnBrowseFileToEncodeButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnEncoderChanged(QString)</slot>
  <slot>OnEncodeButtonClicked()</slot>
  <slot>OnEncodeAsButtonClicked()</slot>
  <slot>OnBrowseFileToEncodeButtonClicked()</slot>
 </slots>
</ui>
